hadoop-hdfs-project/hadoop-hdfs-rbf/src/test/java/org/apache/hadoop/hdfs/server/federation/router/TestConnectionManager.java

```diff
@@ -301,4 +302,15 @@ public void testConfigureConnectionActiveRatio() throws IOException {
 
     tmpConnManager.close();
   }
+
+  @Test
+  public void testUnsupportedProtoExceptionMsg() throws Exception {
+    LambdaTestUtils.intercept(IllegalStateException.class,
+        "Unsupported protocol for connection to NameNode: "
+            + TestConnectionManager.class.getName(),
+        () -> ConnectionPool.newConnection(conf, TEST_NN_ADDRESS, TEST_USER1,
+            TestConnectionManager.class));
+  }
+
+  interface UnsupportedProto { }
```

Comment:
This line now is not required interface UnsupportedProto { } we can remove it.
Apart LGTM